app/src/main/AndroidManifest.xml

```diff
@@ -153,7 +153,9 @@
       android:theme="@style/OppiaThemeWithoutActionBar" />
     <activity
       android:name=".app.topic.TopicActivity"
-      android:theme="@style/OppiaThemeWithoutActionBar" />
+      android:theme="@style/OppiaThemeWithoutActionBar"
+      android:label=""
```

Comment:
@Sparsh1212 we should be setting an actual label here per #2634 (e.g. something that includes the activity title & the current topic open). That will require introducing a new string that supports a parameter (similar to https://github.com/oppia/oppia-android/blob/develop/app/src/main/res/values/strings.xml#L47).

Also, please make sure that you add a new test for this label to make sure it's being properly defined. This may require introducing a new ``TopicActivityTest`` in [this package](https://github.com/oppia/oppia-android/tree/develop/app/src/sharedTest/java/org/oppia/android/app/topic). See [TopicFragmentTest](https://github.com/oppia/oppia-android/blob/develop/app/src/sharedTest/java/org/oppia/android/app/topic/TopicFragmentTest.kt#L91) for an idea on how to properly set up ``TopicActivity`` for testing.